apps/financial-aid/web-veita/src/components/ModalTypes/AcceptModal.tsx

```diff
@@ -310,11 +314,12 @@ const AcceptModal = ({
           value={Number(state.personalTaxCreditPercentage).toString()}
           type="number"
```

Comment:
Should persónuafsláttur be postfixed with '%' or is it clear that this is always a percentage? I wrote a helper hook the other day that did exactly that. Holla if you want to use it 😱

Reply:
I am going to make this nice have and add it to the backlog